runtime/cudaq/operators.h

```diff
@@ -142,10 +147,17 @@ class sum_op {
   std::vector<std::size_t> degrees(bool application_order = true) const;
 
   /// @brief Return the number of operator terms that make up this operator sum.
-  std::size_t num_terms() const;
+  int num_terms() const;
```

Comment:
why the change to int here?

Reply:
I need to do a full pass over int vs size_t usage. I'll come back to it with my take after looking closer at that.

Reply:
using size_t across the board seems to work well. did that.